tracer/build/timeit/Samples.FakeDbCommand/run.cmd

```diff
@@ -3,17 +3,22 @@ IF EXIST results_Samples.FakeDbCommand.windows.net462.json DEL /F results_Sample
 IF EXIST results_Samples.FakeDbCommand.windows.netcoreapp31.json DEL /F results_Samples.FakeDbCommand.windows.netcoreapp31.json
 IF EXIST results_Samples.FakeDbCommand.windows.net60.json DEL /F results_Samples.FakeDbCommand.windows.net60.json
 
+echo *********************
+echo Installing timeitsharp
+echo *********************
+dotnet tool update -g timeitsharp
```

Comment:
wondering if we should specify a version here so that a bad update doesn't break the pipeline for everyone (requires a PR to bump it instead, like we do for other dependencies?)
```suggestion
dotnet tool update -g timeitsharp -version 0.0.8
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3942/commits/17d8311bf60ddddcb92fd37238a55cb2630c940c